infrastructure/src/main/java/org/corfudb/infrastructure/HandlerMethods.java

```diff
@@ -47,14 +47,14 @@
      * on another thread.
      */
     @FunctionalInterface
-    interface Handler<T extends CorfuMsg> {
+    interface HandlerMethod<T extends CorfuMsg> {
         void handle(@Nonnull T corfuMsg,
                     @Nonnull ChannelHandlerContext ctx,
                     @Nonnull IServerRouter r);
     }
 
     /** The handler map. */
-    private final Map<CorfuMsgType, Handler> handlerMap;
+    private final Map<CorfuMsgType, HandlerMethod> handlerMap;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4653599)